it/google-cloud-platform/src/test/java/org/apache/beam/it/gcp/bigquery/BigQueryIOST.java

```diff
@@ -296,11 +297,10 @@ private void generateDataAndWrite(BigQueryIO.Write<byte[]> writeIO) throws IOExc
                 .withSchema(schema)
                 .withCustomGcsTempLocation(ValueProvider.StaticValueProvider.of(tempLocation)));
 
-    String runnerV2Experiment = "use_runner_v2";
     String experiments =
         configuration.writeMethod.equals(STORAGE_API_AT_LEAST_ONCE_METHOD)
-            ? runnerV2Experiment + ",streaming_mode_at_least_once"
-            : runnerV2Experiment;
+            ? GcpOptions.STREAMING_ENGINE_EXPERIMENT + ",streaming_mode_at_least_once"
+            : GcpOptions.STREAMING_ENGINE_EXPERIMENT;
```

Comment:
I think it would be better to leave this on runner v2 since that is ultimately the thing that we want to work better in the long run. If we need to reduce the volume of data in this test, I think that would be better

Reply:
@damccorm The data duplication issue occurs even under small loads specifically in the exactly_once method when using runner_v2. Can we avoid using v2 for this method for now? I also noticed that in [BigQueryIOLT](https://github.com/apache/beam/blob/25977b8b05c103b832d92593602699eb86e7ab04/it/google-cloud-platform/src/test/java/org/apache/beam/it/gcp/bigquery/BigQueryIOLT.java#L188), we explicitly disabled v2, likely for the same reason.

Reply:
> The data duplication issue

What is the specific problem? Is it a correctness problem or does the job just fail?

We should at least file an issue describing the problem when running with runner v2 (even if the problem is not in Beam)

Reply:
In the test, we first count the number of written records inside the write pipeline as a separate PTransform. Then, after the write process is complete, we fetch the row count directly from the BigQuery table and compare these two values. When using the exactly_once method (STORAGE_WRITE_API), the number of records in the BigQuery table should not exceed the value obtained from the write pipeline. However, for some reason, the value in the BigQuery table is higher when using runner_v2. Because of this, the test fails at the assertion of the number of rows.